relationships/candidates/AWSDYNAMODBTABLE.yml

```diff
@@ -12,6 +12,8 @@ lookups:
           field: awsRegion
         - tagKeys: ["aws.tableName"]
           field: awsTableName
+        - tagKeys: ["aws.arn"]
+          field: cloudArn
```

Comment:
I think that the term `cloudArn` is confusing. ARN is an AWS concept, so it doesn't make sense to generalize it as a cloud concept. I'd choose something more specific like `awsArn` or something more generic like `cloudResourceId`.